File: 6/1/Unary.java
class Unary{

	public static void main(String[] args){


		int x = 8;
		int y = 5;
		int ans1 = ++x + x++;
		int ans2 = --y + y--;
		System.out.println(ans1);
		System.out.println(ans2);
		System.out.println(x);
		System.out.println(y);

	}

}

File: 6/2/Unary.c
#include<stdio.h>

int main()
{
	int x = 8;
	int y = 5;
	int ans1 = ++x + x++;
	int ans2 = --y + y--;
	printf("%d\n",ans1);
	printf("%d\n",ans2);
	printf("%d\n",x);
	printf("%d\n",y);
	return(0);	

}